### PR TITLE
Jsx lecture notes

### PR DESCRIPTION
**Pull request by Stan015**
This pull request carefully solved [issue #8 (JSX)](https://github.com/Oluwasetemi/react-note/issues/8) with a well detailed lecture notes on all key aspects of JSX.

### CODE REVIEW

File: pages/jsx.md [hunk 1]
```diff
@@ -8,11 +8,463 @@ hideInToc: true
...
+<>
+  <h1>Making sure all tags are explicitly closed</h1>
+  <p>I have got some good news!</p>
+  <img src="imageUrl" alt="Thank you for closing me properly in JSX" />
+  <input type="text" />
+</>
+```
+
+<div v-click>
+3. Use camelCase for most of the things!: JSX converts to JavaScript, and attributes written in JSX become keys of JavaScript objects. As we know, JavaScript has limitations on variable names, hence using camelCase is recommended. For example, their names can’t contain dashes or be reserved words like `class`. Since `class` is a reserved word, in React you write `className` instead.
```
Comment (Oluwasetemi) [inline pages/jsx.md]: Move no 3 to a separate slide @Stan015. It is cutting out in the preview link.

**Comment by Oluwasetemi** [comment]
@Stan015 the content is great, I did a scan through. Will take a deep look at it later. Let a comment on slide 14.

File: pages/jsx.md (hunk 1 above)
Comment (Stan015) [reply pages/jsx.md]: Done ✅

**Comment by Stan015** [comment]
Hello 👋
Is this pr still expecting more reviews or we are good to move forward?

**Review by Oluwasetemi — APPROVED** [review]

**Comment by Oluwasetemi** [comment]
Merged already. Apologies for the delay

**Pull request merged**